recipes/poco/all/conanfile.py

```diff
@@ -200,6 +202,9 @@ def _configure_cmake(self):
         # On Windows, Poco needs a message (MC) compiler.
         with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
             self._cmake.configure(build_dir=self._build_subfolder)
+        # Disable fork
+        if not self.options.get_safe("enable_fork", True):
+            self._cmake.definitions["POCO_NO_FORK_EXEC"] = True
```

Comment:
```suggestion
        # Disable fork
        self._cmake.definitions["POCO_NO_FORK_EXEC"] = not self.options.get_safe("enable_fork", True)
```

Reply:
Incorporating this suggestion would have been nice, but I won't push since getting this one though c3i is a pain.